src/main/java/ru/practicum/shareit/item/ItemStorageImpl.java

```diff
@@ -0,0 +1,101 @@
+package ru.practicum.shareit.item;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Repository;
+import ru.practicum.shareit.exception.NotFoundException;
+import ru.practicum.shareit.item.dto.ItemDto;
+import ru.practicum.shareit.item.model.Item;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+@Slf4j
+@Repository
+@RequiredArgsConstructor(onConstructor_ = @Autowired)
+public class ItemStorageImpl implements ItemStorage {
+    private final Map<Long, List<Item>> items = new HashMap<>();
+    private long currentId = 1;
+
+    @Override
+    public List<ItemDto> findAll(long userId) {
+        List<Item> userItems = items.get(userId);
+        return userItems.stream().map(ItemMapper::toItemDto).collect(Collectors.toList());
```

Comment:
Для лучшей читаемости стримы принято бить по строкам:
```suggestion
        return userItems.stream()
            .map(ItemMapper::toItemDto)
            .collect(Collectors.toList());
```